src/NuGet.Clients/NuGet.PackageManagement.UI/Resources/Brushes.cs

```diff
@@ -258,5 +259,23 @@ public static void LoadVsBrushes()
             ListItemBackgroundSelectedColorKey = CommonDocumentColors.ListItemBackgroundSelectedColorKey;
             ListItemTextSelectedColorKey = CommonDocumentColors.ListItemTextSelectedColorKey;
         }
+
+        private static bool IsBackgroundColorFlightEnabled() =>
+            ExperimentationService.Default.IsCachedFlightEnabled(ExperimentationConstants.FlightFlags.PackageManagerBackgroundColor)
```

Comment:
[`ExperimentationService.IsCachedFlightEnabled`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.experimentation.experimentationservice.iscachedflightenabled?view=visualstudiosdk-2019#parameters) doc says `flight name is a string no more than 16 characters (case-insensitive)`. Flight name `nuGetPackageManagerBackgroundColor` defined is greater than 16 chars. Will be there be any issues with that ?

Reply:
Thanks for finding that. I'll reduce the size of the string to 16 characters or less to comply with this requirement to be on the safe side.

Reply:
@kartheekp-ms I double checked about the requirement with the experimentation folks who had already signed off on my chosen flight name before I submitted this PR, and it seems the requirement used to be 16 but was updated to 100 characters. I'm going to keep this name as-is.